packages/utilities/src/AutoScroll.ts

```diff
@@ -21,14 +22,16 @@ export class AutoScroll {
   private _events: EventGroup;
   private _scrollableParent: HTMLElement | null;
   private _scrollRect: IRectangle | undefined;
-  private _scrollVelocity: number;
-  private _timeoutId: number;
+  private _VerticalScrollVelocity: number;
```

Comment:
This variable name should probably be lowerCamelCase:

```tsx
private _verticalScrollVelocity: number
```

Reply:
Absolutely. Consider it done.